src/bun.js/javascript.zig

```diff
@@ -1190,6 +1190,10 @@ pub const VirtualMachine = struct {
             }
         }
 
+        if (map.get("NODE_PRESERVE_SYMLINKS")) |_| {
```

Comment:
Does node check the value of the env var? 
```suggestion
        if (map.has("NODE_PRESERVE_SYMLINKS")) {
```